### ISSUE TITLE
[Feature Request] Integrate Overleaf comments

### ISSUE THREAD

**Issue by Fontanapink** [texstudio-org/texstudio#1306]
I LOVE Texstudio and much prefer it over Overleaf. However, my team uses Overleaf to work on projects. I know I can pull changes made in Overleaf to my local repository to see them with Texstudio, but I would love it if I could also view the Overleaf comments that my teammates make. 

Otherwise, for now, I have to have both Texstudio (to edit) and Overleaf (to view pop-up comments) to work in collaborative works. 

### Expected behavior

See Overleaf's comments.

### Actual behavior

Can't see Overleaf team members' comments.

**Comment by sunderme** [comment]
I don't think that it is possible for txs to have access to those comments. Overleaf seems not to offer an interface for that.

**Comment by flhofer** [comment]
I agree. With the pro version, you have access and can clone it as a `git` repository. But even then, the comments are not part of any file present in the repository. I agreed with my team that I simply add my comments as `%TODO:` and `%FIXME:` as used in TexStudio and check-in on overleaf from time to time if there are some new comments from their side.

Actually, it's more likely that you can do the opposite. There are Google-Chrome plugins that act on the text between comments and source on overleaf, [example](https://tex.stackexchange.com/questions/333947/integrating-grammarly-with-online-latex-editors-such-as-overleaf) So it's probably easier to create a plugin that adds a TexStudio-style comment for each overleaf comment.
